kafka-connect-s3/src/main/java/io/confluent/connect/s3/continuum/S3ContinuumConfigValues.java

```diff
@@ -0,0 +1,69 @@
+/*
+ * Copyright 2022 Confluent Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.confluent.connect.s3.continuum;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.ArrayList;
+
+public class S3ContinuumConfigValues {
+  private static final Logger log = LoggerFactory.getLogger(S3ContinuumConfigValues.class);
+
+  public String topic;
+  public int partition;
+  public String bootstrapServers;
+  public String schemaRegistryURL;
+  public String valueConverter;
+
+  public boolean isConfigured() {
+    if (!stringIsNullOrEmpty(this.topic) && !stringIsNullOrEmpty(this.bootstrapServers)) {
```

Comment:
Should we validate if either is set? It seems weird to that include one but not the other will act like neither is set.

```suggestion
    if (!stringIsNullOrEmpty(this.topic) || !stringIsNullOrEmpty(this.bootstrapServers)) {
```